16/umbraco-cms/customizing/property-editors/property-value-converters.md

```diff
@@ -135,17 +156,16 @@ public PropertyCacheLevel GetPropertyCacheLevel(IPublishedPropertyType propertyT
 }
 ```
 
-## Methods - Conversion
-
-There are a few different levels of conversion which can occur.
+## Implement conversion methods
+Implement the methods that perform the conversion from a raw database value to an intermediate value and then to the final type. Conversions happen in two steps.
 
 ### ConvertSourceToIntermediate(IPublishedElement owner, IPublishedPropertyType propertyType, object source, bool preview)
+This method converts the raw data value into an appropriate intermediate type that is needed for the final conversion step to an object. For example, a Content Picker saves the node identifier as a `string`. To return `IPublishedContent`, the final conversion step needs a `Udi` instead. So in the intermediate step, check if the string value is a valid `Udi` and convert the string to a `Udi` as the intermediate value.
```

Comment:
```suggestion
This method converts the raw data value into an appropriate intermediate type that is needed for the final conversion step to an object.

For example:
- A basic text property likely just stores its data as a `string`, so that can be converted to a `string` intermediate value.
- A Content Picker stores the node identifier (`Udi`) as a `string`. To return `IPublishedContent`, the final conversion step needs a `Udi` instead. So in the intermediate step, check if the `string` value is a valid `Udi` and convert the `string` to a `Udi` as the intermediate value.
```